lib/topological_inventory/mock_source/collector.rb

```diff
@@ -184,6 +186,37 @@ def full_refresh_sweep(entity_type, refresh_state_uuid, resource_version, total_
         resource_version
       end
 
+      def save_inventory(collections, refresh_state_uuid = nil, refresh_state_part_uuid = nil)
```

Comment:
I would rather see this in `TopologicalInventoryIngressApiClient::Collector`

Reply:
right, we can move it there later, right now each collector duplicates that. We can probably just pass the `:name => OCP` or whatever

but for now we need this, or the collector will fail

Reply:
yes, I wrote shared collector to remove these duplicates :)

Reply:
right, but we need to change the interface, because `:name => OCP` is OpenShift specific, so can't use used in Amazon

so lets change the interface in client and apply this to all collectors

Reply:
ok, so let's fix this and hopefully "later" we'll modify and use shared code.

Reply:
:+1: yeah, for now lets merge this, so the collector doesn't fail :-)